ofrak_core/ofrak/gui/server.py

```diff
@@ -456,6 +459,23 @@ async def search_for_vaddr(self, request: Request) -> Response:
         except NotFoundError:
             return json_response([])
 
+    @exceptions_to_http(SerializedError)
+    async def add_tag(self, request: Request) -> Response:
+        """
+        Expected POST body is a comment in the form Tuple[Optional[Range], str] (serialized to JSON).
+        """
```

Comment:
```suggestion
```

Does this docstring actually apply, or is it fake news?

Reply:
☝️